### PR TITLE
Test: Fix e2e tests

### PR DESCRIPTION
**Pull request by lmuntanerDfinity**
# Motivation

e2e tests were failing and it was removing the screenshots.

# Changes

* Skip a test that is not ready for CI pipeline. Only locally.
* `OWN_CANISTER_ID` env var is always set.

### CODE REVIEW

**Comment by lmuntanerDfinity** [comment]
@peterpeterparker @max-dfinity please review

Screenshots will be updated automatically by the pipeline with a commit.

File: e2e-tests/specs/example.e2e.ts [hunk 1]
```diff
@@ -27,7 +27,8 @@ describe("landing page", () => {
     await browser["screenshot"]("landing-page");
   });
 
-  it("register and back to dashboard", async () => {
+  // DO NOT RUN ON CI PIPELINE
+  xit("register and back to dashboard", async () => {
```
Comment (peterpeterparker) [inline e2e-tests/specs/example.e2e.ts:31]: it misses a `TODO` or the `xit` should be reverted?

File: frontend/svelte/env.config.mjs [hunk 2]
```diff
@@ -37,8 +37,7 @@ const LEDGER_CANISTER_ID = "ryjl3-tyaaa-aaaaa-aaaba-cai";
 // Testnet canister id
 // TODO: Move to index.html L2-265
 const OWN_CANISTER_ID =
-  process.env.OWN_CANISTER_ID ||
-  (development ? "qhbym-qaaaa-aaaaa-aaafq-cai" : "");
+  process.env.OWN_CANISTER_ID || "qhbym-qaaaa-aaaaa-aaafq-cai";
```
Comment (peterpeterparker) [inline frontend/svelte/env.config.mjs:40]: Is that correct? The production build will no inherit the wrong OWN_CANISTER_ID with this change?

**Review by peterpeterparker — CHANGES_REQUESTED** [review]

File: frontend/svelte/env.config.mjs (hunk 2 above)
Comment (lmuntanerDfinity) [reply frontend/svelte/env.config.mjs:40]: Found the one for prod in the dart project/

File: e2e-tests/specs/example.e2e.ts (hunk 1 above)
Comment (lmuntanerDfinity) [reply e2e-tests/specs/example.e2e.ts:31]: it misses a todo

File: frontend/svelte/env.config.mjs (hunk 2 above)
Comment (peterpeterparker) [reply frontend/svelte/env.config.mjs]: Can you link the reference? When I search for `qhbym-qaaaa-aaaaa-aaafq-cai` I only get `testnet` references 🤔

Comment (lmuntanerDfinity) [reply frontend/svelte/env.config.mjs]: Haha, yuo were too fast. The commit had not yet updated. Check the changes: "qoctq-giaaa-aaaaa-aaaea-cai"

**Review by peterpeterparker — APPROVED** [review]

**Pull request merged**